### PR TITLE
Health regeneration

### PR DESCRIPTION
**Pull request by GBMiro**
Health regeneration fully working.

In HUDEffects.scene you can try this feature along with the HealthLostVisualFeedback and the LowHPWarning features that are in the UI main branch.

You can tweak the regeneration rates in the player script through imgui

### CODE REVIEW

**Comment by jcarlos0305** [comment]
Task linked: [CU-kk6zfz VS3 UI Features](https://app.clickup.com/t/kk6zfz)

**Comment by jcarlos0305** [comment]
Task linked: [CU-kn8rqy Health regeneration](https://app.clickup.com/t/kn8rqy)

**Review by devildrake — APPROVED** [review]

**Review by isaacfdez — APPROVED** [review]

File: Penteract/Assets/Scenes/HealthRegeneration2.scene.meta [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (magalenyo) [inline Penteract/Assets/Scenes/HealthRegeneration2.scene.meta:1]: Delete either HealthRegeneration 1 or 2 and rename

**Review by magalenyo — APPROVED** [review]
Delete unnecessary Scenes

**Pull request merged**